tensorflow/compiler/xla/service/gpu/ir_emission_utils.h

```diff
@@ -254,6 +254,8 @@ int PartitionLmhloOperandsAndOutputs(mlir::Operation* op);
 std::vector<mlir::Value> GetHloOperands(mlir::Operation* op);
 std::vector<mlir::Value> GetHloOutputs(mlir::Operation* op);
 
+// return true if the operation have a row-major memory layout.
```

Comment:
"Returns", "has", "defaults", second sentence is a bit hard to interpret in connection to the first one, should it be "returns whether an instruction has a row-major memory layout (or no layout at all, which defaults to row-major)"?

Also I'm not sure I've seen the term "row-major" before in the codebase, is it present in other places? Does it mean minor-to-major? Maybe would be useful to add a sentence what "row-major" actually means?